qa/tasks/cephfs/filesystem.py

```diff
@@ -362,9 +362,10 @@ class Filesystem(MDSCluster):
     This object is for driving a CephFS filesystem.  The MDS daemons driven by
     MDSCluster may be shared with other Filesystems.
     """
-    def __init__(self, ctx, fscid=None, create=None):
+    def __init__(self, ctx, fscid=None, name='cephfs', create=True):
```

Comment:
@batrick I decided to just redo this interface so it made more sense. Please take a look.